user-info-api/README.md

```diff
@@ -0,0 +1,84 @@
+# API definition for users metadata
+This API-Module contains all data objects, user attributes and claims for users metadata
+
+## Referenced by
+This API is used by the following modules
+* [user-info-provider](./../user-info-provider/README.md)
+* [user-info-requester](./../user-info-requester/README.md)
+
+## Standardized user attributes
+The following attributes are used as an input for this mapper:
+
+|Attribute Key|Type| Description                                                     |
+|:----|:----|:----------------------------------------------------------------|
+|heimatorganisation.bundesland|String| Bundesland                                                      |
+|person.familienname|String| Familienname                                                    |
+|person.vorname|String| Vorname                                                         |
+|person.initialenFamilienname|String| Initial oder Initialen  des Familiennamens                                                  |
+|person.initialenVorname|String| Initial oder Initialen des Vornamens                                                     |
+|person.akronym|String| Akronym                                                         |
+|person.geburtsdatum|String| Geburtsdatum                                                    |
+|person.geschlecht|String| Geschlecht                                                      |
+|person.lokalisierung|String| Lokalisierung                                                   |
+|person.vertrauensstufe|String| Vertrauensstufe                                                 |
+
+### Single context
+|Attribute Key|Type|Description|
+|:----|:----|:----|
+|person.kontext.id|String|ID des Kontexts|
+|person.kontext.org.id|String|ID der Organisation|
+|person.kontext.org.kennung|String|Kenung der Schule (Schulidentifikator)|
+|person.kontext.org.vidis_schulidentifikator|String|Vidis Schulidentifikator|
+|person.kontext.org.name|String|Name der Schule|
+|person.kontext.org.typ|String|Typ der Organisation|
+|person.kontext.rolle|String|Rolle der Person|
+|person.kontext.status|String|Status|
+### Multiple contexts
+
+Context array
+```
+<zero> ::= 0
+<posDigit> ::= 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9
+<digit> ::= <zero> | <posDigit>
+<number> ::= <zero> <posDigit> | <posDigit> <digit>
```

Comment:
What does this mean? :sweat_smile:

Reply:
BNF

Reply:
Aha :smile: no time to understand :runner: 
https://en.wikipedia.org/wiki/Backus%E2%80%93Naur_form